winui/Rating/ToolTip.md

```diff
@@ -1,12 +1,92 @@
 ---
 layout: post
 title: ToolTip with WinUI Rating control | Syncfusion
-description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
+description: Learn here all about Enable tooltip features and tooltip format support in the rating items in WinUI rating control.
 platform: WinUI
 control: SfRating
 documentation: ug
 ---
 
 # ToolTip in WinUI Rating
 
-This section explains the tooltip features available in the WinUI Rating control.
+The Rating control EnableToolTip can be customized using the `EnableToolTip` property. The default value is `false`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfRating
+     Value="3"
+     ItemsCount="5"
+     EnableToolTip="True">
+</syncfusion:SfRating>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfRating rating = new SfRating();
+
+rating.Value = 3;
+rating.ItemsCount = 5;
+rating.EnableToolTip = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Rating control using enabletooltip in WinUI](Rating_images/winui_rating_tooltip.png)
+
+## Rating Control using the ToolTipFormat 
+
+The Rating control ToolTipFormat can be customized using the `ToolTipFormat` property. The default value is `0.##`.
+
```

Comment:
@Priyanka-Vijayasankar The tooltip content can be formatted by using the `ToolTipFormat` property.

Don't mention the default value.

Reply:
Changed the content as mentioned above and remove the default value.